### DISCUSSION TITLE
Wordpress wp-config.php, connection failed

### DISCUSSION THREAD

**Discussion by pavandavide** [planetscale/discussion#423] [Q&A] [answered]
to start a wordpress project I would like to use planetscale as database host, in the wp-config configuration however, also adding "define( 'MYSQL_CLIENT_FLAGS', MYSQLI_CLIENT_SSL );" the connection is not made, are there any solutions?

**Comment by mtolgacoskun** [comment]
> to start a wordpress project I would like to use planetscale as database host, in the wp-config configuration however, also adding "define( 'MYSQL_CLIENT_FLAGS', MYSQLI_CLIENT_SSL );" the connection is not made, are there any solutions?

You can look at this site, it will be exactly the solution to your problem. [https://help.one.com/hc/en-us/articles/115005585929-Update-WordPress-database-connection-details](url)

**Reply by pavandavide** [reply]
Thi solution didnt solves my problem

**Reply by mtolgacoskun** [reply]
Open the config file your WordPress 
Just below the line "define('DB_NAME', 'database_name');", add the following code:

define('MYSQL_CLIENT_FLAGS', MYSQLI_CLIENT_SSL);
define('MYSQL_SSL_CA', '/path/to/ca-cert.pem');
define('MYSQL_SSL_CERT', '/path/to/client-cert.pem');
define('MYSQL_SSL_KEY', '/path/to/client-key.pem');

Specify the path to your SSL/TLS certificates. The MYSQL_SSL_CA constant can be used to specify the path to the certificates.
Note: Make sure that your database server has an SSL/TLS certificate installed, and if it does not support SSL/TLS certificates, this code will not work.

**Comment by JosephShown** [comment, marked as answer]
If you’re having trouble connecting your WordPress site to a PlanetScale database, there are a few things you can try:

Check your connection details: Make sure that you have entered the correct connection details for your PlanetScale database in your wp-config.php file. You can find these details by going to the PlanetScale dashboard, clicking on the branch you want to connect to, and then clicking on the “Connect” button[1](https://planetscale.com/docs/onboarding/connect-to-your-database).

Ensure a secure connection: To ensure a secure connection to your PlanetScale database, you must validate the server-side certificate from PlanetScale. This configuration depends on your application, but often it just means adding a line to your wp-config.php file similar to this:

define('MYSQL_SSL_CA', '/path/to/cert.pem');
The path to the certificate depends on your system. You can find more information about this in the [PlanetScale documentation](https://planetscale.com/docs/tutorials/connect-any-application#option-1-connect-with-username-and-password-recommended)[2](https://planetscale.com/docs/tutorials/connect-any-application).

Use the PlanetScale proxy: Another way to connect your WordPress site to your PlanetScale database during development is by using the PlanetScale proxy. [This allows you to establish a secure connection to PlanetScale without having to configure any credential details](https://planetscale.com/docs/tutorials/connect-any-application)[2](https://planetscale.com/docs/tutorials/connect-any-application).
If you continue to have trouble connecting your WordPress site to your PlanetScale database, it might be worth reaching out to their support team for assistance.
